data/json/items/tool/tailoring.json

```diff
@@ -4,7 +4,7 @@
     "type": "GENERIC",
     "category": "tools",
     "name": "bone sewing awl",
-    "description": "This is a crude sharpened bone awl; those were used for leatherworking before metal was discovered.  It can also serve as an improvised stabbing weapon, but will break quickly.",
+    "description": "This is a crudely sharpened bone awl; those were used for leatherworking before metal was discovered.  It can also serve as an improvised stabbing weapon, but will break quickly.",
```

Comment:
hmm

Reply:
it's either `crude, sharpened` or `crudely sharpened`. I think the former is mostly likely intended.